test/Ionide.ProjInfo.Tests/Tests.fs

```diff
@@ -2303,6 +2303,34 @@ let sample12SlnFilterTest toolsPath loaderType workspaceFactory =
             Expect.equal parsed[0].ProjectFileName fsproj "should contain the expected project"
         )
 
+let sample13SolutionFilesTest toolsPath loaderType workspaceFactory =
+    testCase
+        $"Can load sample13 sln with solution files - {loaderType}"
+        (fun () ->
+
+            let projPath = pathForProject ``sample 13 sln with solution files``
+            let projDir = Path.GetDirectoryName projPath
+
+            let expectedReadme =
+                projDir
+                / "README.md"
+
+            let solutionContents =
+                InspectSln.tryParseSln projPath
+                |> getResult
+
+            let solutionItem = solutionContents.Items[0]
+
+            Expect.equal solutionItem.Guid (Guid("8ec462fd-d22e-90a8-e5ce-7e832ba40c5d")) "Should have the epxcted guid"
+            Expect.equal solutionItem.Name "\\Solution Items\\" "Should have the expected folder name"
```

Comment:
I was going to ask about the slashes here as a separate bug - Ionide is displaying them in the solution explorer and I don't think it should e.g. 

![image](https://github.com/user-attachments/assets/10157a4f-d76a-46a6-9a5d-465ce593cd25)

but then the new slnx format does seem to include them in the actual solution file:

![image](https://github.com/user-attachments/assets/11377115-c1a6-40d2-a6a5-2dc6a6b4e411)

So - not sure if they should be filtered here, or returned to the caller and changed for display purposes higher up?

Reply:
and that's made me realise that Visual Studio displays nested folders as a tree
![image](https://github.com/user-attachments/assets/413231e6-7345-4f3a-b072-5f63fad930c6)

Whereas Ionide displays them alongside each other

![image](https://github.com/user-attachments/assets/f7818c3c-f607-496a-849a-fc46fcc3d357)

I suppose that's a separate bug though

Reply:
IIRC we had some tree support in the past - I wonder if the odd slash usage/naming is throwing that off. Agree that's a separate issue though.

Since VS is stripping the slashes for UI purposes, we could/should as well. The values here almost look like xpath path expressions, except the slashes are the wrong direction...

Reply:
> IIRC we had some tree support in the past

I tried downgrading Ionide to 7.22.0 and the nesting works in that version, so I'll have a look at that next week.

Reply:
Looks like version 0.68.0 returned the folders as a tree, and now it's returning a flat list as it appears in the solution file